lib/util/Consts.ts

```diff
@@ -3,11 +3,17 @@ import type * as RDF from 'rdf-js';
 
 const DF = new DataFactory();
 
-export const TRUE_STR = '"true"^^xsd:boolean';
-export const FALSE_STR = '"false"^^xsd:boolean';
-export const EVB_ERR_STR = '"not an dateTime"^^xsd:dateTime';
+export type LiteralTypes = TypeAlias | TypeURL;
+
+export enum TypeAlias {
+  // Numeric is everything defined in https://www.w3.org/TR/sparql11-query/#operandDataTypes
+  SPARQL_NUMERIC = 'SPARQL_NUMERIC',
+  SPARQL_STRINGLY = 'SPARQL_STRINGLY',
+  SPARQL_OTHER = 'SPARQL_OTHER',
+  SPARQL_NON_LEXICAL = 'SPARQL_NON_LEXICAL',
+}
 
-// TODO: Consider inlining all with 'const enum'
+// TODO: Consider inlining all with 'const enum' - do we need to handle this right now? Can this be removed?
```

Comment:
`const enum` is a TS feature https://www.typescriptlang.org/docs/handbook/enums.html#const-enums that resolves some enum handling at compile time vs runtime. It might be worth to use these for speed, it might not be for other reasons, but we should at least write down why.

Reply:
I'm thinking we don't want this? We probably want to add an extra option to the evaluatorConfig called `additionalExtentionTable` or smth. If we were to add this we need the runtime values. 
It's possible I don't understand const enum  enough but I think it would remove the runtime values.

Reply:
This seems likely.

Reply:
Okey, Should I keep the original `TODO` or can I remove it?

Reply:
As you see fit.

Reply:
I think we can remove it.